thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -806,8 +827,16 @@ class CIndexReadSlaveActivity : public CIndexReadSlaveBase
                                     callback.finishedRow();
                                     return ret.finalizeRowClear(sz);
                                 }
+                                else
+                                {
+                                    ++postFiltered;
+                                }
                             }
                         }
+                        else
+                        {
+                            ++postFiltered;
```

Comment:
as per other trivial/stlye: comment - to be consistent with other code, don't normally put single statement in a curly block.

Reply:
There are some style guides that would say you should use the same curlies (or not) for the if and else branches...